test/metabase/upload_test.clj

```diff
@@ -483,14 +480,20 @@
 (defmacro with-uploads-allowed [& body]
   `(do-with-uploads-allowed (fn [] ~@body)))
 
-(defn- do-with-upload-table! [table thunk]
+(defn do-with-upload-table! [table thunk]
   (try (thunk table)
        (finally
          (driver/drop-table! driver/*driver*
                              (mt/id)
```

Comment:
Can we make this helper more generic by getting the database id from the table?

Reply:
yep 34f5d99a1b54935d8ee509ebb09e132cab2e053f